scripts/bump_cloudbeat.sh

```diff
@@ -253,6 +254,21 @@ bump_snyk_branch_monitoring() {
 
 }
 
-run_version_changes_for_main
+validate_base_branch() {
+    if [[ "$BASE_BRANCH" == "main" || "$BASE_BRANCH" == "8.x" || "$BASE_BRANCH" == "9.x" ]]; then
```

Comment:
In the current state of the repository, the following important branches require version bumps:
- `main`: Currently points to version 9.0.
- `8.x`: The `main` branch for all `"8"` versions.
- `8.*`: Represents all released versions, such as `8.16`, `8.15`, etc.
- `9.*`: Future branches for "9" versions.
To handle these cases, we can verify whether $BASE_BRANCH == "main", additionally, we need to implement a regular expression to match branch patterns for 8. and 9.

Maybe we could consider something like this:
`[[ "$BASE_BRANCH" == "main" || "$BASE_BRANCH" =~ ^(8\..*|9\..*)$ ]]`

Reply:
Since both if statements return the same result, I think it would be better to combine them into a single clause.

Reply:
Done